### DISCUSSION TITLE
Can I Get the Copilot Sidebar Back?

### DISCUSSION THREAD

**Discussion by livejamie** [valinet/ExplorerPatcher#3524] [Ideas]
I understand that Microsoft has removed the Copilot for Windows shortcut to promote its new Copilot+ PCs. I liked the shortcut of pressing a button on my taskbar and having the sidebar slide in allowed me to interact with it and ask questions.

Is this something that Explorer Patcher can reenable?

**Comment by Amrsatrio** [comment]
No.
